app/src/main/java/com/example/admin/lab_56_57/ActivityForLab57.java

```diff
@@ -0,0 +1,45 @@
+package com.example.admin.lab_56_57;
+
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+
+
+public class ActivityForLab57 extends AppCompatActivity {
```

Comment:
No need to rename now, but it's better to use this name: Lab57Activity. In this way it matches naming of MainActivity, for instance.